RoFILib/configuration/Configuration.cpp

```diff
@@ -268,19 +298,77 @@ bool Configuration::removeEdge(const Edge& edge) {
     EdgeList& set1 = edges.at(edge.id1());
     EdgeList& set2 = edges.at(edge.id2());
 
-    int index1 = edge.side1() * 3 + edge.dock1();
-    int index2 = edge.side2() * 3 + edge.dock2();
-    if (!set1[index1].has_value() || !set2[index2].has_value())
+    int setIndex1 = edge.side1() * 3 + edge.dock1();
+    int setIndex2 = edge.side2() * 3 + edge.dock2();
```

Comment:
Momentálně mi uniká proč se to jmenuje "setIndex"

Reply:
Protože se jedná o index do proměnné, která byla pojmneována set (chtěl jsem ještě víc zvýraznit, co to je -- původně jsem se k tomu choval jako k ID).